services/ssp-b/src/index.js

```diff
@@ -47,11 +54,22 @@ app.use((req, res, next) => {
 
 app.use(express.json());
```

Comment:
This line was removed in "services/ssp-a/src/index.js", but here we keep "express", is that correct?